src/bridging/providers/across/createAcrossDepositCall.test.ts

```
@@ -0,0 +1,193 @@
+  type: OrderKind.SELL,
+  sellTokenChainId: SupportedChainId.MAINNET,
+  sellTokenAddress: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
+  sellTokenDecimals: 6,
+  buyTokenChainId: AdditionalTargetChainId.POLYGON,
+  buyTokenAddress: '0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359',
+  buyTokenDecimals: 6,
+  amount: '10000000000',
+  owner: '0x1234567890123456789012345678901234567890',
+}
+
+const ACROSS_QUOTE: AcrossQuoteResult = {
```

ESLint

[error] 'ACROSS_QUOTE' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)